Technotes/001-async-networking.md

```diff
@@ -0,0 +1,246 @@
+---
+date: 2023-03-28
+status: proposed
+authors: Matt Gacy
+---
```

Comment:
YAML front matter; I'm envisioning the possibility of integration with [Jekyll](https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll) in the future